cdk_infra/package.json

```diff
@@ -12,27 +12,27 @@
   "devDependencies": {
     "@types/jest": "^27.5.2",
     "@types/js-yaml": "^4.0.5",
-    "@types/node": "18.11.10",
+    "@types/node": "18.11.18",
     "@types/prettier": "2.7.1",
     "@typescript-eslint/eslint-plugin": "^5.31.0",
     "@typescript-eslint/parser": "^5.31.0",
-    "ajv": "^8.11.0",
+    "ajv": "^8.11.2",
     "ajv-errors": "^3.0.0",
     "aws-cdk": "2.52.0",
-    "eslint": "^8.20.0",
+    "eslint": "^8.31.0",
     "jest": "^27.5.1",
-    "prettier": "^2.7.1",
+    "prettier": "^2.8.1",
     "ts-jest": "^27.1.4",
     "ts-node": "^10.8.1",
     "typescript": "~3.9.7",
     "yaml-schema-validator": "^1.2.3"
   },
   "dependencies": {
     "@aws-cdk/aws-ec2": "^1.161.0",
-    "@aws-cdk/aws-eks": "^1.161.0",
-    "@aws-cdk/aws-iam": "^1.161.0",
-    "@aws-cdk/core": "^1.161.0",
-    "@aws-cdk/lambda-layer-kubectl": "^1.179.0",
+    "@aws-cdk/aws-eks": "^1.182.0",
```

Comment:
Can you break this in two prs? one for updating dependencies and another one with the functionality that you are adding/changing? This will make your PR easier to review.